app/coffee/infrastructure/Mongoose.coffee

```diff
@@ -2,6 +2,8 @@ mongoose = require('mongoose')
 Settings = require 'settings-sharelatex'
 logger = require('logger-sharelatex')
 
+mongoose.Promise = global.Promise
```

Comment:
Yeah, so that they get this promise implementation across the board.

Looking at it again, it makes more sense to do these two things separately as different PRs